site/docs/master/aws-config.md

```diff
@@ -50,87 +51,92 @@ aws s3api create-bucket \
-            "Action": [
-                "s3:GetObject",
-                "s3:DeleteObject",
-                "s3:PutObject",
-                "s3:AbortMultipartUpload",
-                "s3:ListMultipartUploadParts"
-            ],
-            "Resource": [
-                "arn:aws:s3:::${BUCKET}/*"
-            ]
-        },
-        {
-            "Effect": "Allow",
-            "Action": [
-                "s3:ListBucket"
-            ],
-            "Resource": [
-                "arn:aws:s3:::${BUCKET}"
-            ]
-        }
-    ]
-}
-EOF
-```
-```bash
-aws iam put-user-policy \
-  --user-name velero \
-  --policy-name velero \
-  --policy-document file://velero-policy.json
-```
+
+    ```
+    cat > velero-policy.json <<EOF
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "ec2:DescribeVolumes",
+                    "ec2:DescribeSnapshots",
+                    "ec2:CreateTags",
+                    "ec2:CreateVolume",
+                    "ec2:CreateSnapshot",
+                    "ec2:DeleteSnapshot"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:GetObject",
+                    "s3:DeleteObject",
+                    "s3:PutObject",
+                    "s3:AbortMultipartUpload",
+                    "s3:ListMultipartUploadParts"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::${BUCKET}/*"
+                ]
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::${BUCKET}"
+                ]
+            }
+        ]
+    }
+    EOF
+    ```
+    ```bash
+    aws iam put-user-policy \
+      --user-name velero \
+      --policy-name velero \
+      --policy-document file://velero-policy.json
+    ```
 
 3. Create an access key for the user:
-```bash
-aws iam create-access-key --user-name velero
-```
+
+    ```bash
+    aws iam create-access-key --user-name velero
+    ```
 
     The result should look like:
-```json
-{
-  "AccessKey": {
-        "UserName": "velero",
-        "Status": "Active",
-        "CreateDate": "2017-07-31T22:24:41.576Z",
-        "SecretAccessKey": <AWS_SECRET_ACCESS_KEY>,
-        "AccessKeyId": <AWS_ACCESS_KEY_ID>
-  }
-}
-```
+
+    ```json
+    {
+      "AccessKey": {
+            "UserName": "velero",
+            "Status": "Active",
+            "CreateDate": "2017-07-31T22:24:41.576Z",
+            "SecretAccessKey": <AWS_SECRET_ACCESS_KEY>,
+            "AccessKeyId": <AWS_ACCESS_KEY_ID>
+      }
+    }
+    ```
 
 4. Create a Velero-specific credentials file (`credentials-velero`) in your local directory:
-```bash
+
+    ```bash
 [default]
```

Comment:
This doesn't render properly in GH, I believe the content needs to be indented to line up with the backticks